tests/security/cc/ip_eb_tables.pm

```diff
@@ -24,27 +25,33 @@ sub run {
 
     select_console 'root-console';
 
-    # Configure bridge for ip_eb_tables workaround
-    assert_script_run("cat > $f_ifcfg_br0 <<'END'\n$br0_config\nEND\n( exit \$?)");
+    # In system role is CC, the bridge is set up automatically in x86 and arm,
```

Comment:
Change to:
audit ebtables tests needs bridge network, In cc system role....

Reply:
Done